src/libostree/ostree-repo-pull-private.h

```diff
@@ -54,6 +54,10 @@ typedef struct
 
   GVariant *extra_headers;
   char *append_user_agent;
+  long low_speed_limit;
+  long low_speed_time;
```

Comment:
These two should be `guint32` - i.e. explicitly 32 bits because that's what the gvariant type `u` expects.

As is right now, this would cause undefined behavior on 64 bit architectures.